src/tasks/payload/payload_task.c

```diff
@@ -135,7 +135,39 @@ void beacon_down_command_test(slate_t *slate)
     }
 }
 
-void ping_command_test(slate_t *slate)
+bool heartbeat_check(slate_t *slate)
+{
+    const uint64_t TIME_THRESHOLD = 1000;
+    if (slate->is_payload_on)
+    {
+        bool response_received = ping_command_test(slate);
+        uint64_t timeDelta = absolute_time_diff_us(
```

Comment:
Here the unit of measurement is microseconds? We should make sure we're not off by some order of magnitudes.